e2e/config/config.go

```diff
@@ -32,6 +32,9 @@ type Configuration struct {
 	IgnoreScenariosWithMissingVHD bool          `env:"IGNORE_SCENARIOS_WITH_MISSING_VHD"`
 	SkipTestsWithSKUCapacityIssue bool          `env:"SKIP_TESTS_WITH_SKU_CAPACITY_ISSUE"`
 	KeepVMSS                      bool          `env:"KEEP_VMSS"`
+	BlobStorageAccount            string        `env:"BLOB_STORAGE_ACCOUNT" envDefault:"https://abe2e.blob.core.windows.net"`
```

Comment:
is this a new blob storage ? how was this created ? do we have IaC for it? if this is purely to host the binary for the test, I'm wondering if we could avoid that, knowing that the more azure resources we have to more SFI work we have to do.

Reply:
It's a hack to speed e2e feedback loop.

Without it we need to rebuild the VHD before running any tests. It adds about an hour to the loop.
This code put binary on a node and run a similar CSE command to what we're going to use.

I had difficulty finding a way to upload a binary to a VM. It's not available from a public network. So I couldn't SSH into it. And it's not joined k8s cluster yet when we need to execute it.

I think I created it manually (it's been a while ago).

I don't really like and happy to change if you have any other ideas.